src/app/tests/suites/certification/Test_TC_MF_1_30.yaml

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name:
+    22.1.30. [TC-MF-1.30] Open commissioning window - normal operation and
+    closing [ECM] [DUT - Commissionee]
+
+config:
+    nodeId: 0x12344321
+    timeout: 330
+    endpoint: 0
+
+tests:
+    - label: "Factory Reset the DUT_CE"
+      cluster: "SystemCommands"
+      command: "FactoryReset"
+
+    - label: "TH_CR1 starts a commissioning process with DUT_CE"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "TH_CR1 opens a commissioning window on DUT_CE"
```

Comment:
There is a missing step before this one to check that the window is currently reported as closed (which it should be).

If the test plan is missing this, it's an obvious test plan bug.

Reply:
@tcarmelveilleux testplan is missing this step i will raise an issue for testplan regarding this.